maven-artifact/src/test/java/org/apache/maven/artifact/versioning/ComparableVersionTest.java

```diff
@@ -46,13 +46,16 @@ private Comparable newComparable(String version) {
     }
 
     private static final String[] VERSIONS_QUALIFIER = {
+        "1-abc",
```

Comment:
So this one will be interpreted as alpha?

Reply:
lexically abc < alpha